clients/docs/java.rst

```diff
@@ -0,0 +1,317 @@
+      Producing record: alice {"count":6}
+      Producing record: alice {"count":7}
+      Producing record: alice {"count":8}
+      Producing record: alice {"count":9}
+      Produced record to topic test1 partition [0] @ offset 0
+      Produced record to topic test1 partition [0] @ offset 1
+      Produced record to topic test1 partition [0] @ offset 2
+      Produced record to topic test1 partition [0] @ offset 3
+      Produced record to topic test1 partition [0] @ offset 4
+      Produced record to topic test1 partition [0] @ offset 5
+      Produced record to topic test1 partition [0] @ offset 6
+      Produced record to topic test1 partition [0] @ offset 7
+      Produced record to topic test1 partition [0] @ offset 8
+      Produced record to topic test1 partition [0] @ offset 9
+      10 messages were produced to topic test1
+      ...
+
+#. View the :devx-examples:`producer code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ProducerExample.java`.
+
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the consumer, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka cluster
+   - the topic name you used earlier
+
+   .. code-block:: text
+
+      # Compile the Java code
+      mvn clean package
+
+      # Run the consumer
+      mvn exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.ConsumerExample" \
+      -Dexec.args="$HOME/.confluent/java.config test1"
+
+#. Verify the consumer received all the messages. You should see:
+
+   .. code-block:: bash
+
+      ...
+      Consumed record with key alice and value {"count":0}, and updated total count to 0
+      Consumed record with key alice and value {"count":1}, and updated total count to 1
+      Consumed record with key alice and value {"count":2}, and updated total count to 3
+      Consumed record with key alice and value {"count":3}, and updated total count to 6
+      Consumed record with key alice and value {"count":4}, and updated total count to 10
+      Consumed record with key alice and value {"count":5}, and updated total count to 15
+      Consumed record with key alice and value {"count":6}, and updated total count to 21
+      Consumed record with key alice and value {"count":7}, and updated total count to 28
+      Consumed record with key alice and value {"count":8}, and updated total count to 36
+      Consumed record with key alice and value {"count":9}, and updated total count to 45
+
+#. When you are done, press ``Ctrl-C``.
+
+#. View the :devx-examples:`consumer code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ConsumerExample.java`.
+
+
+Kafka Streams
+~~~~~~~~~~~~~
+
+#. Run the Kafka Streams application, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka
+     cluster
+   - the topic name you used earlier
+
+   .. code-block:: text
+
+      # Compile the Java code
```

Comment:
Next 3 lines are now eligible for deletion (assuming you add the distinct compile step above)